tests/BOAClient.test.ts

```diff
@@ -424,7 +424,7 @@ export class TestStoa {
                 if (Number.isNaN(height)) height = enrolled_height;
 
                 for (let elem of sample_validators) {
-                    elem.preimage.distance = height - enrolled_height;
+                    elem.preimage.height = (height - enrolled_height).toString();
```

Comment:
```suggestion
                    elem.preimage.height = height.toString();
```
The suggested code seems semantically right because the `height` of a pre-image does not mean a distance anymore.